go.work

```diff
@@ -7,6 +7,7 @@ use (
 	./lib/fixtures
 	./lib/flattening
 	./lib/ocrypto
+	// ./lib/fqn
```

Comment:
```suggestion
	./lib/fqn
```

Reply:
@ryanulit @dmihalcik-virtru It looks like the image build action changes when this is added to the `go.work`. I'm guessing it's because the module is not yet merged?

Reply:
Updated CI to fix and pushed up a commit including this in go.work